05-classification.Rmd

```diff
@@ -47,16 +48,18 @@ prediction are called a **training set**. We call them a "training set" because
 we use these observations to train, or teach, our classifier so that we can use
 it to make predictions on new data that we have not seen previously.
 
-There are many possible classification algorithms that we could use to predict
-a categorical class/label for an observation. In addition, there are many
-variations on the basic classification problem, e.g.,  binary classification
-where only two classes are involved (e.g. disease or healthy patient), or
-multiclass classification, which involves assigning an object to one of several
-classes (e.g., private, public, or not for-profit organization). Here we will
-focus on the simple, widely used **K-nearest neighbours** algorithm for the
-binary classification problem. Other examples you may encounter in future
-courses include decision trees, support vector machines (SVMs), logistic
-regression, and neural networks.
+There are many possible methods that we could use to predict
+a categorical class/label for an observation. In this book we will
+focus on the simple, widely used **K-nearest neighbours** algorithm.
+In your future studies, you might encounter decision trees, support vector machines (SVMs),
+logistic regression, neural networks, and more; see the additional resources
+section at the end of the next chapter for where to begin learning more about
+these other methods. It is also worth mentioning that there are many
+variations on the basic classification problem. For example, 
+we focus on the setting of **binary classification** where only two
+classes are involved (e.g., a diagnosis of either healthy or diseased), but you may 
+also run into multiclass classification problems with more than two
+categories (e.g., a diagnosis of healthy, bronchitis, pneumonia, or a common cold).
```

Comment:
However, in the worksheets and tutorial problems sets we do have multi-class classification problems... Which makes me wonder, should we mention them in the book?

Reply:
I would like to include it, but it would require us to do one of:

- have multiple datasets in this chapter (because the one we have is no good for multiclass)
- rewrite the chapter for a dataset where multiclass is doable
- put a paragraph at the end (or somewhere else) without actually exploring it at all

I think the only feasible one of these to do is the third -- I'll open an issue for us to address it separately from this PR.

Reply:
Issue #163 is open now